lms/djangoapps/grades/signals/handlers.py

```diff
@@ -57,6 +57,9 @@ def submissions_score_set_handler(sender, **kwargs):  # pylint: disable=unused-a
     user = user_by_anonymous_id(kwargs['anonymous_user_id'])
     if user is None:
         return
+    if points_possible == 0:
```

Comment:
Keep in mind this outstanding ticket: https://openedx.atlassian.net/browse/TNL-5982
Essentially, the grading subsystem expects all scorable problems to have positive `max_scores` (a.k.a., possible points). 
Since this issue is specific to ORA, I agree with isolating this change in the submissions handler code (and not emitting any events).